packages/@okta/vuepress-site/docs/reference/api/templates/index.md

```diff
@@ -371,13 +371,13 @@ All templates have the following properties:
 | template               | Text of the template, including any [macros](#sms-template-macros). | String (See note below)                                        | FALSE    | 1         | 161       |
 | created                | Timestamp when template was created                                 | String (ISO-8601)                                              | TRUE     | N/A       | N/A       |
 | lastUpdated            | Timestamp when template was last updated                            | String (ISO-8601)                                              | TRUE     | N/A       | N/A       |
-| translations           | Array of [translations](#translation-attributes)                    | Array                                                          | N/A      | N/A       | N/A       |
+| translations           | A key-value map of [translations](#translation-attributes)                    | Translations Object                                                          | N/A      | N/A       | N/A       |
 
 > **Note:** The final length of your SMS message cannot exceed 160 characters. If the verification code portion of the message falls outside of the 160-character limit, your message will not be sent.
 
 #### Translation Attributes
```

Comment:
Is it a must for this heading to include the word "Attributes" (presumably for consistency with similar headings such as "SMS Attributes" above), or would it be OK for this heading to be just "Translations"?  In the table above we refer to this as "translations", and even though the href leads to the right place (good), it's still a little mental leap that ideally shouldn't be there.

Reply:
Our model is that, for every object, we have an "[Object] Properties" section that tells you what's inside that object.